runtime/kusama/src/lib.rs

```diff
@@ -1607,6 +1730,16 @@ pub mod migrations {
 		frame_support::migrations::RemovePallet<TipsPalletName, <Runtime as frame_system::Config>::DbWeight>,
 
 	);
+
+	// Upgrade Session keys to include BEEFY key.
+	// When this is removed, should also remove `OldSessionKeys`.
+	pub struct UpgradeSessionKeys;
+	impl frame_support::traits::OnRuntimeUpgrade for UpgradeSessionKeys {
+		fn on_runtime_upgrade() -> Weight {
+			Session::upgrade_keys::<OldSessionKeys, _>(transform_session_keys);
+			Perbill::from_percent(50) * BlockWeights::get().max_block
```

Comment:
```suggestion
			BlockWeights::get().max_block
```

50% sounds rather random. Just let's take all the weight.